core/chat/server.go

```diff
@@ -178,8 +179,10 @@ func (s *server) sendWelcomeMessageToClient(c *Client) {
 		// Add an artificial delay so people notice this message come in.
 		time.Sleep(7 * time.Second)
 
-		initialChatMessageText := fmt.Sprintf("Welcome to %s! %s", data.GetServerName(), data.GetServerSummary())
-		initialMessage := models.ChatEvent{ClientID: "owncast-server", Author: data.GetServerName(), Body: initialChatMessageText, ID: "initial-message-1", MessageType: "SYSTEM", Visible: true, Timestamp: time.Now()}
-		c.write(initialMessage)
+		welcomeMessage := data.GetServerWelcomeMessage()
```

Comment:
Should we trim spaces from the entire message?

Reply:
Good idea, I'll add this.